mithril-aggregator/src/http_server/routes/artifact_routes/cardano_database.rs

```diff
@@ -75,6 +79,7 @@ mod handlers {
 
     /// List artifacts
     pub async fn list_artifacts(
+        _origin_tag: Option<String>,
```

Comment:
If we don't use it in the handler, we can remove the middleware for this route? Same comment for all other occurrences.